tests/forte/notebooks/notebook_test_tutorial.py

```diff
@@ -0,0 +1,39 @@
+from testbook import testbook
+
+
+@testbook("docs/notebook_tutorial/handling_structued_data.ipynb")
```

Comment:
These look like actual test to me, don't need to name this file as "tutorial"

Reply:
The actual test needed for current notebooks is simply running through them. In this script, it shows different ways of testing.

Reply:
I mean, is this a test that will be run when `coverage run -m pytest tests/forte/notebooks`? If so, the file name don't have to be `notebook_test_tutorial.py`, it should probably be a name referring the actual test (like structure_data_test or somthing).

Reply:
OK. I think it's not intended to be running by `pytest`. Since the test of `handling_structued_data.ipynb` is included in `handling_structued_data_test.py`. Other tests under `notebook_test_tutorial.py` is unrelated to notebooks.